CHANGELOG.md

```diff
@@ -31,11 +31,12 @@ Main (unreleased)
 
 - Added OpenTelemetry logs and metrics support to Alloy mixin's dashboards and alerts. (@thampiotr)
 
-v1.7.1
------------------
-
```

Comment:
Moving this entry as the change "better handling of truncated queries" was not included in 1.7.1.